### DISCUSSION TITLE
I can't get my email verified

### DISCUSSION THREAD

**Discussion by sarvam207** [community/community#147886] [GitHub Education] [answered]
### Hi everyone,

My application is getting rejected stating that the email I provide must be issued from the school but the email provided is already the one provided by the university. why am I still seeing this pop up and what should I do?

**Comment by davevad93** [comment, marked as answer]
Hi @sarvam207,

Make sure your academic email has been verified in your GitHub account, and if you haven't already done so, set your academic email as your primary email and try submitting your application again.

If after doing this you still get rejected, try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem.

**Reply by sarvam207** [reply]
Thanks it worked!
